### ISSUE TITLE
Setting formControl to FormGroup

### ISSUE THREAD

**Issue by hansip** [ngx-formly/ngx-formly#2016]
**Description**
In my custom formly template, I used `prePopulate` hook to set the `formControl` to be a `FormGroup`. However, on any value update, it raised an error
`Cannot read property 'length' of undefined`. I checked that the error is thrown from `FormlyForm.trackModelChanges`, specifically the code
```javascript
this.modelChangeSubs.push(valueChanges.subscribe((value) => {
  // workaround for https://github.com/angular/angular/issues/13792
  if ((control as any)._onChange.length > 1) {...
```
Looking at angular source code, `_onChange` is an internal variable only on `FormControl`, but not in `FormGroup`, and my custom formly template control is a FormGroup.

**Minimal Reproduction**

https://stackblitz.com/edit/ngx-formly-custom-template-pumeug?file=app%2Fformly-field-input.ts

**Your Environment**

- Angular version: 8.2.14
- Formly version: 5.5.8

**Comment by aitboudad** [comment]
💡you may use the following workaround until fixing the issue in our side:

```ts
// temporary workaround for https://github.com/ngx-formly/ngx-formly/issues/2016
field.formControl['_onChange'] = [];
```

**Comment by hansip** [comment]
Excellent. Thanks a lot @aitboudad

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.5.9](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.5.9) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by hansip** [comment]
Works great @aitboudad thanks for the quick response 👍
